### ISSUE TITLE
Inaccurate Teleporting

### ISSUE THREAD

**Issue by myleshart301** [jhuckaby/magic-sorting-system#27]
My teleportation is not very accurate. It'll sometimes teleport a block in front of the item frame and miss the hopper. I tried to put blocks in front of it, but then the items just disappear. About a third of the items end up on the chest or on the floor instead of on top of the hopper. I'm using the auto chest setup

**Comment by Brendzt** [comment]
im not the creator, but ive put item frame directly on top of the hopper. and ive had no issue with it.

**Comment by Knito58** [comment]
Although I try to lock up every escape direction of the items with glass blocks most of the wool gets routed "back up". I tried to block "up" with another block but no, it insists. Only about a 3rd of the items find the hopper below, the rest is cumulating as bulk-item somewhere on an edge.

This is on newest 1.16 snapshot with magig-sorting-system-2.0.1

**Comment by Knito58** [comment]
> 
> 
> im not the creator, but ive put item frame directly on top of the hopper. and ive had no issue with it.

Did you put it horizontally on the hopper? How would you know what is inside the chest? Do you use ordinary signs for that then?

**Comment by Brendzt** [comment]
@Knito58 
sorry for late reply but this is how i have them set up, ive had no issues with them like this. When the hopper is full the items just sets on top of the hopper then goes in when there is space, No items lost so far.
![2020-06-25_20 44 01](https://user-images.githubusercontent.com/39664381/85811878-d183c180-b724-11ea-90cc-c555afa24476.png)

**Comment by myleshart301** [comment]
Ya but then u kinda lose the bonus of it labeling ur chests as well

**Comment by Knito58** [comment]
@Brendzt Thank you. I will do that like you did. @myleshart301 youre totally right but there r signs and I can jump and meanwhile I'm just happy that it works.

**Comment by jhuckaby** [comment]
@Brendzt That is a very clever solution!  Awesome!  I'll write this up in the new version and give you full credit!

**Comment by Brendzt** [comment]
@Knito58 Yes i put signs directly on the chest, doesn't bother me to have to find the right pixel to open the chest from there. @jhuckaby  Thank you! when i saw your original pictures showing the item frames on the wall with the hoppers below, I was thinking that it would just be easier to on them directly on top for the items can stack into the hoppers.

**Comment by isaaclepes** [comment]
I am using the frames on hoppers and have had some misses.  I tracked this down to volume of items and setup a rig with a hopper feeding a dropper with a comparator back to the dropper so it drops any item it receives.  This lowers the flow-rate significantly and prevents overflowing the hopper at the teleport destination.

**Comment by Greaper88** [comment]
Yeah, been doing the mounting to top of hoppers for quite awhile myself. Knito58's problem was caused by the fact that glass block act as item elevators... regular solid blocks work well though in that situation. The "bonus" of using the item frames as the identification of what's in the chest has never been a thing for me since my storage systems tend to be VERY tall. lol

**Comment by Knito58** [comment]
Thank you @Greaper88. I wanted to see what happened and didn't remember the elevator attribute of glass blocks. Curiosity killed the cat.
